univention/udm_user.py

```diff
@@ -0,0 +1,514 @@
+- udm_user: name=FooBar
+            password=secure_password
+            firstname=Foo
+            lastname=Bar
+'''
+
+
+RETURN = '''# '''
+
+
+def main():
+    expiry = date.strftime(date.today()+relativedelta(years=1), "%Y-%m-%d")
+    module = AnsibleModule(
+        argument_spec = dict(
+            birthday                = dict(default=None,
+                                           type='str'),
+            city                    = dict(default=None,
+                                           type='str'),
+            country                 = dict(default=None,
+                                           type='str'),
+            departmentNumber        = dict(default=None,
+                                           type='str'),
+            description             = dict(default=None,
+                                           type='str'),
+            displayName             = dict(default=None,
+                                           type='str'),
+            email                   = dict(default=[''],
+                                           type='list'),
+            employeeNumber          = dict(default=None,
+                                           type='str'),
+            employeeType            = dict(default=None,
+                                           type='str'),
+            firstname               = dict(default=None,
+                                           type='str'),
+            gecos                   = dict(default=None,
+                                           type='str'),
+            groups                  = dict(default=[],
+                                           type='list'),
+            homeShare               = dict(default=None,
+                                           type='str'),
+            homeSharePath           = dict(default=None,
+                                           type='str'),
+            homeTelephoneNumber     = dict(default=[],
+                                           type='list'),
+            homedrive               = dict(default=None,
+                                           type='str'),
+            lastname                = dict(default=None,
+                                           type='str'),
+            mailAlternativeAddress  = dict(default=[],
+                                           type='list'),
+            mailHomeServer          = dict(default=None,
+                                           type='str'),
+            mailPrimaryAddress      = dict(default=None,
+                                           type='str'),
+            mobileTelephoneNumber   = dict(default=[],
+                                           type='list'),
+            organisation            = dict(default=None,
+                                           type='str'),
+            pagerTelephonenumber    = dict(default=[],
+                                           type='list'),
+            password                = dict(default=None,
```

Comment:
For password / secret arguments `no_log=True` should be set

See http://docs.ansible.com/ansible/developing_modules.html#module-checklist for details